packages/app/src/utils/networks.ts

```diff
@@ -153,6 +156,18 @@ export const NETWORKS: Record<NETWORK, Network> = {
     shortName: "plasma",
     nativeAsset: NATIVE_ASSET.XPL,
   },
+  [NETWORK.ZETACHAIN_TESTNET]: {
+    chainId: NETWORK.ZETACHAIN_TESTNET,
+    name: "zetachain",
+    shortName: "zetachain-testnet",
+    nativeAsset: NATIVE_ASSET.ZETA,
+  },
+  [NETWORK.ZETACHAIN]: {
+    chainId: NETWORK.ZETACHAIN,
+    name: "zetachain",
+    shortName: "zetachain",
```

Comment:
```suggestion
    shortName: "zetachain-mainnet",
```
https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-7000.json#L19